erizo_controller/erizoJS/models/SessionDescription.js

```diff
@@ -205,23 +205,34 @@ class SessionDescription {
     sdp.setOrigin({
       username: '-',
       sessionId: 0,
-      sessionVersion: 0,
+      sessionVersion: info.getSessionVersion(),
       netType: 'IN',
       ipVer: 4,
       address: '127.0.0.1' });
     sdp.name = 'LicodeMCU';
 
     sdp.msidSemantic = { semantic: 'WMS', token: generateRandom(10) };
 
-    if (info.hasAudio()) {
-      const media = getMediaInfoFromDescription(info, sdp, 'audio');
-      sdp.addMedia(media);
+    if (info.getFirstMediaReceived() === 'audio') {
```

Comment:
I see the problem, but I prefer a less hacky solution... would it be possible to add medias in the same order they appear in the bundle group field?

Reply:
Yeah seems a little hacky way also to me but unfortunately the datas are fetched from SdpInfo as separate field (audio and video).
Until there's a refactor in Sdpinfo.cpp that support an ordered array of media (that I tought will be done in the future with the singlepeerconnection feature) I can't think other ways to pass the order than this hacky variable.